lib/v2/chinathinktanks/viewpoint.js

```diff
@@ -0,0 +1,60 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const timezone = require('@/utils/timezone');
+
+module.exports = async (ctx) => {
+    const link = `https://www.chinathinktanks.org.cn/content/list?id=${ctx.params.id}&pt=1`;
+
+    const response = await got(link);
+    const $ = cheerio.load(response.data);
+    const title = $('#relates > div.pub_right > div.location').text().split('：')[1].trim();
+
+    const list = $('#relates > div.pub_right > div.pub_content > ul > li')
+        .map(function () {
+            const info = {
+                title: $(this).find('a').attr('title'),
+                link: `https:${$(this).find('a').attr('href')}`,
+                pubdate: $(this).find('span').text().slice(1, -1).trim(),
+            };
+            return info;
+        })
+        .get();
+
+    const out = await Promise.all(
+        list.map(async (info) => {
+            const title = info.title;
+            const pubdate = info.pubdate;
+            const itemUrl = info.link;
+
+            const cache = await ctx.cache.get(itemUrl);
+            if (cache) {
+                return Promise.resolve(JSON.parse(cache));
+            }
```

Comment:
Please use `ctx.cache.tryGet` instead

Reply:
You can take reference from the previous PR #8778
https://github.com/DIYgod/RSSHub/blob/63ba1db4f63c327a32a8c530cee0549617d435ef/lib/v2/cool18/index.js#L34-L54